package-lock.json

```diff
@@ -3137,7 +3137,7 @@
     },
     "next-tick": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/next-tick/-/next-tick-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/next-tick/-/next-tick-1.0.0.tgz",
```

Comment:
Why do your commits always change these lines?
Can you run `mp build` in your spec and then `mp upgrade` to see if these links are `https`?

Reply:
I pulled develop, ran mp update and I guess this appeared as a result.